README.md

```diff
@@ -66,6 +66,31 @@ pip install -r requirements.txt
 
 This will install the required Python packages in a virtual environment.
 
+## Resource Group
+
+This module requires an existing Azure Resource Group. You can either:
+
+1. Create one with Terraform:
+
+  ```hcl
+  resource "azurerm_resource_group" "materialize" {
+    name     = "${var.prefix}-rg"
```

Comment:
Should we just have people use the `resource_group_name` variable?
https://github.com/MaterializeInc/terraform-azurerm-materialize/blob/main/variables.tf#L11

Reply:
that is, since `resource_group_name` is a variable w/o a default -- they can just set `resource_group_name` in  a tfvars file for either case ...